pegasus/sites.v3/code.org/public/teacher-dashboard/sections.haml

```diff
@@ -155,3 +156,5 @@ content-type: text/ng-template
   %br
     %p= I18n.t('dashboard_privacy_document_link')
   %br/
+
+%script{src: 'https://apis.google.com/js/api.js'}
```

Comment:
https://www.npmjs.com/package/googleapis Could we have this be part of our package instead?